site/docs/identity/guides/settingUpIdP.mdx

```diff
@@ -0,0 +1,52 @@
+---
+id: settingUpIdP
+title: Configuring Your Identity Provider
+slug: /identity/guides/settingUpIdP
+description: Guide on how to set up IdP
+keywords:
+  - bandwidth
+  - identity
+  - identity provider
+  - setup
+hide_title: true
+image: ../../static/img/bandwidth-logo.png
+---
+## Configuring Your Identity Provider
+
+Configuring your identity provider for Bandwidth will require you have a few pieces of information ready. You will need
+
+### Information we need from you
+
+Our service needs information about your identity provider in order to know whom to trust about your users' information and where to go to try to initiate single sign-on.
+
+- `usernameField` - This indicates which field in your SAML assertion represents the username of the user signing in. This value must match the username in our system.
+By default, this will be using the `NameId` of the `Subject` in the assertion.
+- `issuerUri` - this is the Entity ID of your SAML Identity Provider which will send the SAML assertion back to us upon a successful sign-in.
+- `singleSignOnUrl` - the SAML endpoint on your Identity Provider that we will `POST` to in order to initiate an authentication on behalf of a user.
+	- NOTE: we only support a `POST` binding when initiating SSO.
+- `signatureCertificate` - either a PEM or DER decoded certificate to verify the authenticity of your SAML Identity Provider.
+
+### Information you need from us
+
+Your identity provider will need information about our service provider to know whom to trust and whither to send information.
```

Comment:
whither -> whether

Reply:
I meant "whither" in the sense of "to where"

Reply:
People who don't know that word should simply go read some Tolkien